src/main/java/build/buildfarm/instance/memory/MemoryInstance.java

```diff
@@ -331,7 +332,7 @@ protected void validateAction(Action action) {
       Duration maximum = config.getMaximumActionTimeout();
       Preconditions.checkState(
           timeout.getSeconds() < maximum.getSeconds() ||
-          (timeout.getSeconds() == maximum.getSeconds() && timeout.getNanos() < maximum.getNanos()));
+              (timeout.getSeconds() == maximum.getSeconds() && timeout.getNanos() < maximum.getNanos()));
```

Comment:
I'm surprised that this got indented. regardless, it isn't correct style as it was before. nothing to do here